### ISSUE TITLE
EF 2.0

### ISSUE THREAD

**Issue by ARZ-SagA** [reinhard-brandstaedter/dynatrace-plugin-exasol#3]
Any plans to migrate this into Extension Framework 2.0? :-)

**Comment by Det27** [comment]
Yeah this would be great and highly appreciated when this extension/plugin gets migrated to EF 2.0. According to this article: https://www.dynatrace.com/news/blog/unmatched-scalability-and-security-of-dynatrace-extensions-now-available-for-all-supported-technologies-7-reasons-to-migrate-your-jmx-and-python-plugins/
there is a how-to guide available. But I was not able to find it.
@reinhard-brandstaedter: Could you let us know if a migration to EF 2.0 is planned? Is there a How-To Guide we can use to migrate this plugin to the EF 2.0? Thank you so much for your support!
